packages/patternfly-4/react-core/src/demos/PaginationTable.md

```diff
@@ -48,21 +73,101 @@ class ComplexPaginationTableDemo extends React.Component {
       />
     );
   }
+  renderTable() {
+    const { err, loading, res } = this.state;
+    if (loading) {
+      return (
+        <Table
+          cells={['Title', 'Body']}
+          rows={[
+            {
+              heightAuto: true,
+              cells: [{
+                props: {colSpan: 8},
+                title: (
+                  <Bullseye>
+                    <EmptyState variant={EmptyStateVariant.small}>
+                      <EmptyStateIcon variant="container" component={Spinner} />
+                      <Title size="lg">Loading</Title>
+                    </EmptyState>
+                  </Bullseye>
+                )
+              }]
+            }
+          ]}
+        >
+          <TableHeader />
+          <TableBody />
+        </Table>
+      );
+    }
+    if (err) {
+      return (
+        <Table
+          cells={['Title', 'Body']}
+          rows={[
+            {
+              heightAuto: true,
+              cells: [{
+                props: {colSpan: 8},
+                title: (
+                  <Bullseye>
+                    <EmptyState variant={EmptyStateVariant.small}>
+                      <EmptyStateIcon icon={TimesIcon} />
+                      <Title size="lg">Failed to fetch data</Title>
+                    </EmptyState>
+                  </Bullseye>
```

Comment:
I'm not able to see this empty state. How does the user see an empty state with this demo?

Perhaps we can add a toggle?